diagnostic.md

```diff
@@ -51,19 +51,19 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
-
+It is an absolute path, because the path starts with Users and goes down the path to the Desktop directory.
  <!-- Answer Ends Here -->
 
  2. Suppose that we're working on a project, and we want to use a font that's being hosted on Google at `https://fonts.googleapis.com/css?family=Open+Sans`. Is that an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
-
+It is an relative path because we are referencing a link that is being remotely accessed.
```

Comment:
This is an absolute path since "http://..." refers to the same place no matter which location the file is in.